eva-accession-import/src/test/resources/input-files/fasta/fastaWithSoftMasking.fa.fai

```diff
@@ -0,0 +1 @@
+NW_006738765.1	160	128	80	81
```

Comment:
Not sure I understand. So the FAI file was included when it shouldn't have been?